LICENSE

```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Shouldn't we keep the licence? AFAIK it's the same we're using in other projects.

Reply:
Do you mean an Apache license? Probably, though I it shouldn't be a problem to use GPL, since it's a wrapper, not modifications on existing software.

Reply:
In web we are using GPLv3 AFAIK, never really checked all repos. I just noticed here because you removed it altogether!

Reply:
Ah. Will recover it.